lib/create-icon-set.js

```diff
@@ -38,11 +43,13 @@ export default function createIconSet(glyphMap, fontFamily, fontFile) {
       color: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
       children: PropTypes.node,
       style: PropTypes.any, // eslint-disable-line react/forbid-prop-types
+      extraStyle: PropTypes.any, // eslint-disable-line react/forbid-prop-types
```

Comment:
What is this `extraStyle` used for? Can't see it referenced elsewhere.

Reply:
This was part of an earlier solution which I seemed to have missed when cleaning up the code, on it!